core-bundle/tests/Fixtures/Functional/Routing/localhost.yml

```diff
@@ -3,11 +3,11 @@ tl_page:
       pid: 0
       sorting: 128
       tstamp: 1539696285
-      title: Domain with port
-      alias: domain-with-port
+      title: Localhost
+      alias: localhost
       type: root
       language: en
-      dns: domain-with-port.local:8080
+      dns: 127.0.0.1
```

Comment:
Just curious - why did you change to an IP here?

Reply:
Because that is the most common use case in combination with a port number. The corresponding functional test uses the URL `https://127.0.0.1:8080` as e.g. the Symfony local web server does.